lib/commands/build.js

```diff
@@ -23,9 +25,23 @@ module.exports = Command.extend({
       analytics: this.analytics,
       project: this.project
     });
+    var ShowAssetSizesTask = this.tasks.ShowAssetSizes;
+    var showTask = new ShowAssetSizesTask({
+      ui: this.ui
+    });
 
-    return win.checkWindowsElevation(this.ui).then(function() {
-      return buildTask.run(commandOptions);
+    return win.checkWindowsElevation(this.ui).then(function () {
+      return Promise.resolve()
```

Comment:
I recall that was because of `.then()` being undefined on `buildTask.run` within the tests. I misunderstood the usage of `stub`. I'm now providing `Promise.resolve()` as an argument to `stub`, so this should be resolved.